pending/itunes.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+	Rules in CleanerML written/updated by https://github.com/Tobias-B-Besemer
+	v0.2.0 done on 2019-03-07
+
+-->
+<cleaner id="itunes" os="windows">
+  <label>iTunes</label>
+  <description>Apple All-in-one Software</description>
+  <option id="itunes_win8.1">
+    <label>iTunes till Win8.1</label>
+    <description>Delete the iTunes files on Windows till version 8.1</description>
+    <action command="delete" search="glob" path="%AppData%\Apple Computer\iTunes\*.log"/>
+    <action command="delete" search="glob" path="%AppData%\Apple Computer\iTunes\*.ipcc"/>
+    <action command="delete" search="glob" path="%AppData%\Apple Computer\iTunes\*.ipsw"/>
+    <action command="delete" search="walk.all" path="%AppData%\Apple Computer\Logs\"/>
+    <action command="delete" search="walk.all" path="%AppData%\Apple Computer\Logs\CrashReporter\MobileDevice\"/>
+    <action command="delete" search="glob" path="%AppData%\Apple Computer\Logs\CrashReporter\MobileDevice\"/>
```

Comment:
This is a glob without a pattern

Reply:
Sorry, my English... You mean why I use a glob ??? The first one, or the second, or both?

I have in my docu https://github.com/Tobias-B-Besemer/docs/blob/cleanerml-examples/_posts/cleanerml-examples.md that for *.XYZ I need to use a glob, because the others don't work as excepted. The second is to remove the folder - we figured that out together on a other example.